src/main/java/gyro/google/compute/NetworkResource.java

```diff
@@ -109,12 +111,22 @@ public void setId(String id) {
         this.id = id;
     }
 
+    @Output
+    public String getSelfLink() {
+        return selfLink;
+    }
+
+    public void setSelfLink(String selfLink) {
+        this.selfLink = selfLink;
+    }
+
```

Comment:
This isn't related to PR.
This should be part of a different PR stating the need to add this.

Reply:
Sure I'll remove this as it's already covered in #51